api/scpca_portal/utils.py

```diff
@@ -50,3 +56,15 @@ def filter_dict_list_by_keys(list_of_dicts, included_keys, *, ignore_value_error
         new_list_of_dicts.append(dictionary_copy)
 
     return new_list_of_dicts
+
+
+def write_dicts_to_tsv(
+    list_of_dicts: List[Dict],
+    output_file_name: str,
+    field_names: Set,
+) -> None:
+    """Writes a list of dictionaries to a csv-like file (exact delimiter provided)."""
+    with open(output_file_name, "w", newline="") as raw_file:
+        csv_writer = csv.DictWriter(raw_file, fieldnames=field_names, delimiter=common.TAB)
+        csv_writer.writeheader()
+        csv_writer.writerows(list_of_dicts)
```

Comment:
So I think we really just want sensible application wide defaults in this method as it is primarily syntactic sugar. This will allow us to keep implementation dry but retain the power of the underlying `csv.DictWriter` command. For example if we want to specify `N/A` for `restval` we could do that in this method for everything, but if we have a use case that requires empty strings we would still be able to supply that.

```suggestion
def write_dicts_to_file(
    list_of_dicts: List[Dict],
    output_file_name: str,
    **kwargs,
) -> None:
    kwargs["fieldnames"] = kwargs.get("fieldnames", ...)
    kwargs["delimeter"] = kwargs.get("delimiter", common.TAB)
    with open(output_file_name, "w", newline="") as raw_file:
        csv_writer = csv.DictWriter(raw_file, **kwargs)
        csv_writer.writeheader()
        csv_writer.writerows(list_of_dicts)
```

Reply:
Added.